scout/log/log.py

```diff
@@ -0,0 +1,63 @@
+import os
+import sys
+import logging
+
+def init_log(logger, filename=None, loglevel=None):
```

Comment:
Yeah :sweat_smile:, I'm still a little confused about this whole implicit inheritance works and what really is a root logger to some other type of logger, hehe. But what you say make sense - keep up the good work!